### PR TITLE
Updating Symantec proxy package

### PR DESCRIPTION
**Pull request by v-sabiraj**
Required items, please complete
   
   Change(s):
   - Updated the data connector instructions by adding steps to configure custom string format in the bluecoat console.
   - Optimized the parser by removing regex and processing the logs with simple parse statement

   Reason for Change(s):
   - Updated the data connector instructions with custom string format and parser as part of icM 400271015

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

image

### CODE REVIEW

**Comment by vakohl** [comment]
@v-sabiraj Please help updating the details of this changes section.

File: Solutions/SymantecProxySG/Parsers/SymantecProxySG/SymantecProxySG.txt [hunk 1]
```diff
@@ -20,42 +20,9 @@
...
-    virus_id = tostring(Parser3[1]), 
-    app_name = tostring(Parser3[2]), 
-    app_operation = tostring(Parser3[3]), 
-    src_port = tostring(Parser3[4]), 
-    cs_threat_risk = tostring(Parser3[6]),
-    country = replace(@"%20",@' ',tostring(Parser3[5]))
-| project-away Parser, Parser2, Parser3, RemainingString1, RemainingString2
+| parse SyslogMessage with logTime:datetime " " time_taken:long " " c_ip:string " " cs_userdn:string " " cs_auth_groups:string " " exception_id " " sc_filter_result:string ' "' cs_categories:string '" "' cs_referrer:string '" ' sc_status:int " " s_action:string " " cs_method:string ' "' content_type '" ' cs_uri_scheme:string " " cs_host:string " " cs_uri_port:int " " cs_uri_path:string " " cs_uri_query:string " " cs_uri_extension:string ' '  Part2:string
+| parse Part2 with '"' UserAgent '" ' Part3
+| extend cs_user_agent = iff(Part2 startswith "-", "-", UserAgent),
```
Comment (vakohl) [inline Solutions/SymantecProxySG/Parsers/SymantecProxySG/SymantecProxySG.txt:25]: @v-sabiraj 
cs_user_agent field is coming as null in new parser where values show when old parser used. I found one but there could more like these. Please re-validate all the fields in new parser. No field should come as empty in new parser if the same show value with old parser.
Also, we would need to test all Analytics rules and Workbook part of this solution by validating that they are not giving any errors when used with the new parser.

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Solutions/SymantecProxySG/Parsers/SymantecProxySG/SymantecProxySG.txt (hunk 1 above)
Comment (v-mchatla) [reply Solutions/SymantecProxySG/Parsers/SymantecProxySG/SymantecProxySG.txt:25]: Hi Varun,
We have tested the parser with customer data and its working as expected. 
image

Thanks

**Comment by v-mchatla** [comment]
Hi @vakohl 
I have made all the changes you have suggested and tested all the content and they are just working as expected.
Could you please provide your approval.
Thanks

**Comment by vakohl** [comment]
> Hi @vakohl I have made all the changes you have suggested and tested all the content and they are just working as expected. Could you please provide your approval. Thanks

@v-mchatla Thankyou Meena. Parser change is approved. Please have data connector change validated with team.

**Review by v-atulyadav — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Review by vakohl — APPROVED** [review]

**Pull request merged**